tools/pepsi/src/wireless.rs

```diff
@@ -61,6 +68,20 @@ async fn status(naos: Vec<NaoAddress>) {
     .await;
 }
 
+async fn scan(naos: Vec<NaoAddress>) {
+    ProgressIndicator::map_tasks(
+        naos,
+        "Starting network scan...",
+        |nao_address, _progress_bar| async move {
+            let nao = Nao::try_new_with_ping(nao_address.ip).await?;
+            nao.scan_networks()
+                .await
+                .wrap_err_with(|| format!("failed to get network status from {nao_address}"))
```

Comment:
I think you mixed up this message with the one below

Reply:
Whoopsie, fixed now.